webrtc.html

```diff
@@ -5426,10 +5429,10 @@ <h4>Processing Remote MediaStreamTracks</h4>
             </p>
           </li>
           <li>
-            <p>Add a new <code><a>RTCTrackEvent</a></code> with
+            <p>Create a new object with
```

Comment:
If the RTCTrackEvent constructor above is addressed, this should say create a new RTCTrackEventInit. You might have to say trackEventInits is an empty list of RTCTrackEventInits rather than an empty list?

Reply:
Yes, I like that!

Reply:
@jan-ivar that is really what I wanted to do, but is it allowed? I somehow got the impression that dictionaries are not "real" things, but can be used in the process of creating something "real" such as an object. If we can use dictionaries in this way I'll change immediately!

Reply:
I believe so. At least the WebIDL bindings in the Firefox codebase lets us do this. This is necessary for instance to satisfy APIs that return dictionaries.

As to supporting links, I found https://github.com/heycam/webidl/issues/142 which is asking for a more compact syntax for initializing dictionaries from prose, so I assume from that that doing such a thing in the first place is expected. :)